scripts/examples/.env.bootnode.example

```diff
@@ -5,3 +5,6 @@ BOOTNODES=enode://e21e2053005e40653d055fc01a07768357749d27d630702c203b1a3c00bdf2
 
 # in case `sudo` is needed
 PERMISSION_PREFIX=
+
+# Set to true to run on spark testnet
+TESTNET=false
```

Comment:
I wonder if we need this as that's the default..
I prefer to hid that config from validators. So they don't do `TESTNET=true` for some reason

Reply:
Yes might be a good idea to avoid confusion